### PR TITLE
SLM-186 updates

### PR DESCRIPTION
**Pull request by mikehalmamoj**

### CODE REVIEW

File: gradlew [hunk 1]
```diff
@@ -1,7 +1,7 @@
-#!/usr/bin/env sh
+#!/bin/sh
 
 #
-# Copyright 2015 the original author or authors.
+# Copyright © 2015-2021 the original authors.
```
Comment (smcveigh941) [inline gradlew:4]: 2022? :)

File: gradlew.bat [hunk 2]
```diff
@@ -1,89 +1,89 @@
...
-rem the _cmd.exe /c_ return code!
-if  not "" == "%GRADLE_EXIT_CONSOLE%" exit 1
-exit /b 1
-
-:mainEnd
-if "%OS%"=="Windows_NT" endlocal
-
-:omega
+@rem
+@rem Copyright 2015 the original author or authors.
```
Comment (smcveigh941) [inline gradlew.bat:2]: Do you want to do the same copyright dates here?

File: gradlew (hunk 1 above)
Comment (mikehalmamoj) [reply gradlew:4]: I've just updated gradle - not responsible for their code 😁

**Review by smcveigh941 — APPROVED** [review]
LGTM

**Pull request merged**